inference-engine/src/vpu/graph_transformer/src/middleend/pass_manager.cpp

```diff
@@ -250,7 +240,18 @@ PassSet::Ptr PassManager::buildMiddleEnd() {
 
     ADD_PASS(swConvAdaptation);
     ADD_PASS(swDeconvAdaptation);
+
+    //
+    // Replace Global AvgPooling with ReduceMean
+    //
+    // this stage should be executed after "hwPoolTiling"
+    // and before "swPoolAdaptation"
```

Comment:
It's already better, but I'd happy to see more descriptive explanation. Why it should be so? Usually it's because your pass depend on some actions of other passes (e.g. dependency produce special stage types or kind of that).